### PR TITLE
[ADD] l10n_de_location_nuts addon

### PR DESCRIPTION
**Pull request by antespi**
This addon depends on oca/partner-contact/base_location_nuts, merge pending https://github.com/OCA/partner-contact/pull/107
# NUTS Regions for Germany

This module allows to import NUTS locations for Germany.
- German states (NUTS level 2) as Partner State
- German districts (NUTS level 3) as Partner Substate
- German regions (NUTS level 4) as Partner Region

After installation, you must click at import wizard to populate NUTS items
in Odoo database in:
Sales > Configuration > Address Book > Import NUTS 2013

This wizard will download from Europe RAMON service the metadata to
build NUTS in Odoo. This localization addon will inherit this wizard and
relate each German NUTS item with German state.

Only Administrator can manage NUTS list (it is not neccesary because
it is an European convention) but any registered user can read them,
in order to allow to assign them to partner object.

### CODE REVIEW

**Comment by OSguard** [comment]
we need to add https://github.com/OCA/partner-contact to .travis file

**Comment by antespi** [comment]
Please, review OCA/partner-contact#107 (and if it is correct, merge). This addon depends on base_location_nuts. That's why travis and runbot fail

**Comment by antespi** [comment]
runbot error is because this PR OCA/partner-contact#107 is not meged yet

**Comment by rafaelbn** [comment]
Waiting review of https://github.com/OCA/partner-contact/pull/107

**Comment by rafaelbn** [comment]
Hi, who is the responsible of German localization?

**Comment by OSguard** [comment]
me - i just restart runbot & travis build

**Comment by rafaelbn** [comment]
Thanks @OSguard !

**Comment by OSguard** [comment]
@rafaelbn please add the oca_dependencies.txt
like here: https://github.com/OCA/rma/pull/63/files to get CI green

**Comment by antespi** [comment]
@rafaelbn @OSguard rebased and oca_dependencies.txt added
Hope this fix travis :smile:

**Comment by OSguard** [comment]
@guewen can you help us, what do we wrong?

**Comment by antespi** [comment]
This will not work properly until https://github.com/OCA/partner-contact/issues/233 is merged

**Comment by rafaelbn** [comment]
Hi,

This module is ready to be merged. I have been using in production enviroment for 3 months

:+1: 

cc @pedrobaeza

File: .travis.yml [hunk 1]
```diff
@@ -22,6 +22,8 @@ virtualenv:
   system_site_packages: true
 
 install:
+  - pip install requests
+  - git clone https://github.com/OCA/partner-contact ${HOME}/partner-contact -b ${VERSION}
```
Comment (pedrobaeza) [inline .travis.yml]: Put this in oca_dependencies.txt

Comment (pedrobaeza) [inline .travis.yml]: I see it's already put, so remove it from here.

**Comment by pedrobaeza** [comment]
Little comment, but :+1: @OSguard, if you give your OK, I'll fix the comment when merging.

**Comment by OSguard** [comment]
:+1: do it

**Comment by antespi** [comment]
Rebase and partner-contact dependency removed from travis file

**Comment by pedrobaeza** [comment]
We can merge this one also.

**Pull request merged**